src/utils/github-api.js

```diff
@@ -6,6 +6,10 @@ exports.getPullRequestByBranch = function(options, callback) {
   });
 };
 
+exports.getClosedPullRequests = function(options, callback) {
+  json_request.get(apiUrl(options, 'pulls?state=closed&per_page=1000'), {auth: options.auth}, callback);
```

Comment:
Do we really need that many? It's a fairly beefy response. The API also [specifies a max of 100?](https://developer.github.com/v3/#pagination). It's reverse-chron sorting, so should be plenty to grab the latest closed PRs.